packages/dotnet-sdk/src/TeamsFx/Microsoft.TeamsFx.csproj

```diff
@@ -31,17 +30,8 @@
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Graph" Version="4.5.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="Microsoft.Graph" Version="4.0.0" />
```

Comment:
Why downgrade this package?

Reply:
I met an error when integrated with sample codes, sample codes hasn't upgraded this package and is using 4.0.0.